nservicebus/azure-service-bus/brokered-message-creation.md

```diff
@@ -25,7 +25,7 @@ NOTE: Both the sender (native or NServiceBus) and the receiver must have a consi
 
 By default, the `BrokeredMessage` body is assumed to be a byte array to remain backward compatible with previous versions of the transport. It is recommended that the body can be retrieved using `Stream` for new implementations, as it incurs less of a memory overhead during serialization. It is especially useful for scenarios, such as native integration, as most non .NET ASB SDK's only support the `Stream` body format.
 
-NOTE: The `SupportedBrokeredMessageBodyTypes.Stream` body format is recommended, but this mode is currently NOT compatible with ServiceControl.
+NOTE: The `SupportedBrokeredMessageBodyTypes.Stream` body format is recommended, but this mode is currently NOT compatible with ServiceControl. A [ServiceControl transport adapter](/servicecontrol/transport-adapter/) is required in order to leverage both.
```

Comment:
@mikeminutillo my understanding was that users need to create such an adapter themselves. There is an issue next in line in platdev to make a bootstrapper to make it simpler,https://github.com/Particular/PlatformDevelopment/issues/1288, is that what you're referring too?